build.gradle

```diff
@@ -100,12 +100,12 @@ subprojects {
 
     checkstyleMain {
         maxWarnings = 0
-        source sourceSets.main.output.resourcesDir
+        source = sourceSets.main.output.resourcesDir
```

Comment:
Heh. I got burned by the same recommendation in a Kotlin project a few months ago.  That's when I decided to switch the Gradle buildscripts in that project from Groovy to Kotlin because the IntelliJ inspector no longer misreported this recommendation.  I chalked it up to the static nature of Kotlin having less surprises than the dynamic nature of Groovy.  For example, if, when setting a property, you call it like a method:

```
something value
```

instead of setting it like a property:

```
something = value
```

The Groovy DSL has a handler that maps an "unknown method" call like the former into a property setter like the latter.  IMO, that seems to just encourage sloppy programming.